corehq/apps/fixtures/utils.py

```diff
@@ -62,3 +64,11 @@ def get_index_schema_node(fixture_id, attrs_to_index):
 def clear_fixture_cache(domain):
     from corehq.apps.fixtures.models import FIXTURE_BUCKET
     get_blob_db().delete(key=FIXTURE_BUCKET + '/' + domain)
+
+
+@task(serializer='pickle', queue='background_queue')
```

Comment:
does this need 'pickle' serialization?

Reply:
Just used the settings we used everywhere. updating to use default itself.